usecases/schema/class.go

```diff
@@ -201,8 +201,12 @@ func setDefaultQuantization(vectorIndexType string, vectorIndexConfig schemaConf
 			hnswConfig.PQ.Enabled = true
 		case "sq":
 			hnswConfig.SQ.Enabled = true
-		case "rq":
+		case "rq-1":
 			hnswConfig.RQ.Enabled = true
+			hnswConfig.RQ.Bits = 1
```

Comment:
If we set the Bits at this stage do we overwrite the default RescoreLimit as well?